### PR TITLE
(examples) Fix link in with-passport example README

### LINKED ISSUE #11961: Passport.js Auth example has 404 link
# Bug report
https://github.com/zeit/next.js/tree/canary/examples/with-passport
One link from the example above gives 404 error
## Describe the bug

`A DB is not included. You can use any db you want and add it **here**.`

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to https://github.com/zeit/next.js/tree/canary/examples/with-passport
2. Click on "here" (https://github.com/zeit/next.js/blob/canary/lib/user.js)

## Expected behavior

Redirect to correct url

## Screenshots

image

### PR DESCRIPTION
**Pull request by jamesmosier**
This closes #11961

### CODE REVIEW

File: examples/with-passport/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This example show how to use [Passport.js](http://www.passportjs.org) with Next.
 
 The example shows how to do a login, signup and logout; and to get the user info using a hook with [SWR](https://swr.now.sh).
 
-A DB is not included. You can use any db you want and add it [here](/lib/user.js).
+A DB is not included. You can use any db you want and add it [here](https://github.com/zeit/next.js/blob/canary/examples/with-passport/lib/user.js).
```
Comment (lfades) [inline examples/with-passport/README.md:7]: I think the following works too:

```suggestion
A DB is not included. You can use any db you want and add it [here](lib/user.js).
```

Comment (jamesmosier) [reply examples/with-passport/README.md:7]: Cool I'll test that out really quick. I tried a few different variations but not that. Thanks!

Comment (jamesmosier) [reply examples/with-passport/README.md]: Yep that did it! Worked on my fork. Thanks 🙏

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.8s | ⚠️ +126ms |
| nodeModulesSize | 55.6 MB | 55.6 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.204 | 2.123 | -0.08 |
| / avg req/sec | 1134.06 | 1177.51 | ⚠️ +43.45 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.274 | 1.281 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 1962.99 | 1951.93 | -11.06 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5e5e.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.7 kB | 56.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.85 kB | 6.85 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| index.html gzip | 929 B | 929 B | ✓ |
| link.html gzip | 939 B | 939 B | ✓ |
| withRouter.html gzip | 927 B | 927 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| buildDuration | 13.4s | 13.9s | ⚠️ +533ms |
| nodeModulesSize | 55.6 MB | 55.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5e5e.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.7 kB | 56.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.85 kB | 6.85 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _error.js | 558 kB | 558 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 558 kB | 558 kB | ✓ |
| link.js | 595 kB | 595 kB | ✓ |
| routerDirect.js | 587 kB | 587 kB | ✓ |
| withRouter.js | 587 kB | 587 kB | ✓ |
| Overall change | 2.89 MB | 2.89 MB | ✓ |

</details>

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| buildDuration | 12.1s | 12s | -135ms |
| nodeModulesSize | 55.6 MB | 55.6 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.986 | 2.031 | ⚠️ +0.05 |
| / avg req/sec | 1258.58 | 1231.05 | -27.53 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.028 | 0.984 | -0.04 |
| /error-in-render avg req/sec | 2432.41 | 2541.87 | ⚠️ +109.46 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5e5e.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.7 kB | 56.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.85 kB | 6.85 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| index.html gzip | 929 B | 929 B | ✓ |
| link.html gzip | 939 B | 939 B | ✓ |
| withRouter.html gzip | 927 B | 927 B | ✓ |
| Overall change | 2.79 kB | 2.79 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| buildDuration | 12.7s | 12.6s | -75ms |
| nodeModulesSize | 55.6 MB | 55.6 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.25 kB | 6.25 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..5e5e.js gzip | 10.3 kB | 10.3 kB | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.a1..NSE.txt gzip | 220 B | 220 B | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 56.7 kB | 56.7 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 4.79 kB | 4.79 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| de003c3a9d30..dule.js gzip | 6.85 kB | 6.85 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| Overall change | 51.4 kB | 51.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.2 kB | 26.2 kB | ✓ |
| Overall change | 26.2 kB | 26.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.js gzip | 1.23 kB | 1.23 kB | ✓ |
| _error.js gzip | 3.12 kB | 3.12 kB | ✓ |
| hooks.js gzip | 663 B | 663 B | ✓ |
| index.js gzip | 222 B | 222 B | ✓ |
| link.js gzip | 2.06 kB | 2.06 kB | ✓ |
| routerDirect.js gzip | 280 B | 280 B | ✓ |
| withRouter.js gzip | 278 B | 278 B | ✓ |
| Overall change | 7.85 kB | 7.85 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _app.module.js gzip | 598 B | 598 B | ✓ |
| _error.module.js gzip | 2.09 kB | 2.09 kB | ✓ |
| hooks.module.js gzip | 383 B | 383 B | ✓ |
| index.module.js gzip | 223 B | 223 B | ✓ |
| link.module.js gzip | 1.52 kB | 1.52 kB | ✓ |
| routerDirect..dule.js gzip | 279 B | 279 B | ✓ |
| withRouter.m..dule.js gzip | 278 B | 278 B | ✓ |
| Overall change | 5.37 kB | 5.37 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 61 B | 61 B | ✓ |
| _buildManife..dule.js gzip | 61 B | 61 B | ✓ |
| Overall change | 122 B | 122 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | zeit/next.js canary  | jamesmosier/next.js fix/issue-11961 | Change |
| - | - | - | - |
| _error.js | 558 kB | 558 kB | ✓ |
| 404.html | 4.18 kB | 4.18 kB | ✓ |
| hooks.html | 3.81 kB | 3.81 kB | ✓ |
| index.js | 558 kB | 558 kB | ✓ |
| link.js | 595 kB | 595 kB | ✓ |
| routerDirect.js | 587 kB | 587 kB | ✓ |
| withRouter.js | 587 kB | 587 kB | ✓ |
| Overall change | 2.89 MB | 2.89 MB | ✓ |

</details>

</details>

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Thanks!

**Pull request merged**